beebop/poppunkWrapper.py

```diff
@@ -99,21 +104,22 @@ def create_microreact(self, cluster: str, internal_cluster: str) -> None:
             model_dir=self.db_fs.db,
             previous_clustering=self.db_fs.previous_clustering,
             previous_query_clustering=(
-                self.fs.previous_query_clustering(self.p_hash)),
+                self.fs.previous_query_clustering(self.p_hash)
+            ),
             previous_mst=None,
             previous_distances=None,
-            network_file=self.fs.network_file(self.p_hash),
+            network_file=None,
```

Comment:
Is this not needed?

Reply:
sorry yes it is i reverted... i was just testing with a dev poppunk branch where ncik is removing